src/test/java/com/github/martapietka/ppm2pbm/rgbToVvalueTest.java

```diff
@@ -0,0 +1,86 @@
+package com.github.martapietka.ppm2pbm;
+
+import static org.assertj.core.api.Assertions.*;
+import org.junit.jupiter.api.Test;
+
+public class rgbToVvalueTest {
```

Comment:
Tu też wielka litera. Poza tym same testy są bardzo dobre, potem Ci pokażę jak można to ujednolicić, ale na razie zostaw tak i śmiało działaj.

Pytanie czy chcemy zostać przy konwersji do V z HSV, do przemyślenia, ale na teraz to dobry punkt wyjścia.